src/renderer/atlas/AtlasEngine.api.cpp

```diff
@@ -695,30 +704,35 @@ void AtlasEngine::_resolveFontMetrics(const wchar_t* requestedFaceName, const Fo
     {
         std::wstring fontName{ requestedFaceName };
         const auto fontWeightU16 = gsl::narrow_cast<u16>(requestedWeight);
-        const auto underlinePosU16 = gsl::narrow_cast<u16>(underlinePos);
-        const auto underlineWidthU16 = gsl::narrow_cast<u16>(underlineWidth);
-        const auto strikethroughPosU16 = gsl::narrow_cast<u16>(strikethroughPos);
-        const auto strikethroughWidthU16 = gsl::narrow_cast<u16>(strikethroughWidth);
-        const auto doubleUnderlinePosTopU16 = gsl::narrow_cast<u16>(doubleUnderlinePosTop);
-        const auto doubleUnderlinePosBottomU16 = gsl::narrow_cast<u16>(doubleUnderlinePosBottom);
-        const auto thinLineWidthU16 = gsl::narrow_cast<u16>(thinLineWidth);
+        const auto baselineU16 = static_cast<u16>(baseline + 0.5f);
+        const auto descenderU16 = gsl::narrow_cast<u16>(cellHeight - baselineU16);
+        const auto underlinePosU16 = static_cast<u16>(underlinePos + 0.5f);
+        const auto underlineWidthU16 = static_cast<u16>(underlineWidth + 0.5f);
+        const auto strikethroughPosU16 = static_cast<u16>(strikethroughPos + 0.5f);
+        const auto strikethroughWidthU16 = static_cast<u16>(strikethroughWidth + 0.5f);
+        const auto doubleUnderlinePosTopU16 = static_cast<u16>(doubleUnderlinePosTop + 0.5f);
+        const auto doubleUnderlinePosBottomU16 = static_cast<u16>(doubleUnderlinePosBottom + 0.5f);
+        const auto thinLineWidthU16 = static_cast<u16>(thinLineWidth + 0.5f);

```

Comment:
`lrintf` rounds halfway cases to nearest instead of away from 0, so for instance 2.5 will be rounded to 2 instead of 3. I don't want this to occur here, so I'm doing a classic, standard +0.5f & cast to integer. I don't think the previous code was always correct without this +0.5f.